### ISSUE TITLE
Build trigger fails while using token authorization

### ISSUE THREAD

**Issue by ddimitrioglo** [jenkinsci/generic-webhook-trigger-plugin#38]
When I'm trying to invoke an endpoint without `user:token`, just by using `?token=xxxx` the build is triggered but getting the error:

```
{
    "status": "ok",
    "data": {
        "triggerResults": {
            "jenkins-test": "Exception occurred, full stack trace in Jenkins server log. Thrown in: org.jenkinsci.plugins.gwt.GenericTriggerResults:21"
        }
    }
}
```

P.S. If I provide a wrong token the given response has right error message.

**Comment by tomasbjerre** [comment]
Probably same as https://github.com/jenkinsci/generic-webhook-trigger-plugin/issues/37

An update should fix it

**Comment by ddimitrioglo** [comment]
@tomasbjerre thank you.

FYI: If I use filter, the filtered repo gives expected data:
```
{
    "status": "ok",
    "data": {
        "triggerResults": {
            "jenkins-test": {
                "id": 0,
                "regexpFilterExpression": ".*jenkins-test$",
                "regexpFilterText": "$test",
                "searchUrl": "",
                "triggered": false,
                "url": ""
            },
            "test-jenkins": "Exception occurred, full stack trace in Jenkins server log. Thrown in: org.jenkinsci.plugins.gwt.GenericTriggerResults:21"
        }
    }
}
```

**Comment by ddimitrioglo** [comment]
@tomasbjerre Confirm that was fixed in 1.26. Thanks!
